README.md

```diff
@@ -52,6 +52,30 @@ Checks if given arguments are all `Arrays`.
 ```
 <!-- prettier-ignore-end -->
 
+#### chunk
+
+Returns the new array of chunks.
```

Comment:
Please rephrase it to make it more clear for humans to get what the function really does. Please see for example https://hackage.haskell.org/package/split-0.2.3.3/docs/Data-List-Split.html#v:chunksOf and https://lodash.com/docs/4.17.15#chunk